src/main/java/seedu/address/logic/parser/ParserUtil.java

```diff
@@ -217,6 +217,14 @@ public static String parseSingleWordFromFindCommand(String searchString) throws
         return trimmedArgs;
     }
 
+    /**
+     * Parses a string containing two dates, validates the format, and returns
+     * an array of two {@code LocalDate} objects, representing the start and end dates.
+     *
+     * @param searchString The input string containing the date range in the format "startDate:endDate".
+     * @return An array of two {@code LocalDate} objects: the start date and end date.
+     * @throws ParseException If the input format is invalid or if the start date is after the end date.
+     */
     public static LocalDate[] parseAttendanceDate(String searchString) throws ParseException {
```

Comment:
maybe a more intuitive name could be parseAttendanceDateRange?

Reply:
yepp will change this